api/config/lighthouse.php

```diff
@@ -136,8 +136,8 @@
     */
 
     'security' => [
-        'max_query_complexity' => GraphQL\Validator\Rules\QueryComplexity::DISABLED,
-        'max_query_depth' => GraphQL\Validator\Rules\QueryDepth::DISABLED,
+        'max_query_complexity' => 50000, // examples: applicant dashboard - 508, candidate search - 2491, AvailablePoolsToAddTo - 27001, ScreeningAndEvaluation_Candidates - 4201, PoolsInFilter - 20001
+        'max_query_depth' => 20, // examples: applicant dashboard - 6
```

Comment:
may be 10? 20 looks extremely high. But we should also do better logging when this fails so we know what we hit.